scripts/core/main.cpp

```diff
@@ -1,8 +1,44 @@
+#include "Config.h"
+#include "ProcessManager.h"
+#include "Utils.h"
 #include <iostream>
+#include <windows.h>
+#include <fcntl.h>
+#include <io.h>
 
-int main() {
+BOOL WINAPI CtrlHandler(DWORD fdwCtrlType) {
+    switch (fdwCtrlType) {
+    case CTRL_CLOSE_EVENT:
+        Logger::Log(L"Console close event detected, ignoring to keep process running in background.", Logger::Level::WARN);
+        return TRUE;
+    case CTRL_C_EVENT:
+    case CTRL_BREAK_EVENT:
+        return TRUE;
+    case CTRL_LOGOFF_EVENT:
+    case CTRL_SHUTDOWN_EVENT:
+    default:
+        return FALSE;
+    }
 }
+int main() {
+    _setmode(_fileno(stdout), _O_U16TEXT); // ignore warnings, they stink
+    if (!SetConsoleCtrlHandler(CtrlHandler, TRUE)) {
+        Logger::Log(L"FATAL ERROR: Could not set console control handler.", Logger::Level::ERROR);
+        Sleep(5000);
+        return 1;
+    }
+    HWND console_window = GetConsoleWindow();
+    Config config;
+    if (!config.load(".\\config\\config.ini")) {
```

Comment:
I prefer a more solid way of finding the configuration. It would be great to put the path of the config into a header file maybe.

```suggestion
    if (!config.load(PATH_CONFIGURATION)) {
```